arcade/application.py

```diff
@@ -930,6 +931,19 @@ def on_mouse_leave(self, x: int, y: int):
         """
         pass
 
+    def save_screenshot(self, location: Optional[str] = None) -> Path:
```

Comment:
1. I think `location` would be less ambiguous if renamed as `path` to avoid ambiguity about screen location
2. It may be a good idea to support `Union[Path, str]` here instead